extensions/wikia/CommunityPage/CommunityPageSpecialHooks.class.php

```diff
@@ -73,7 +73,7 @@ public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 			\Wikia::addAssetsToOutput( 'community_page_benefits_scss' );
 		}
 
-		if ( !$out->getUser()->isAnon() ) {
+		if ( !$out->getUser()->isAnon() && !$out->getUser()->isAllowed( 'first-edit-dialog-exempt' )) {
```

Comment:
I'd suggest putting `$out->getUser()` in a variable since it's used three times in this method now. Also, missing a space before the closing parenthesis.